ext/date/tests/009.phpt

```diff
@@ -3,7 +3,6 @@ strftime() and gmstrftime() tests
 --SKIPIF--
 <?php
 if (substr(PHP_OS, 0, 3) == 'WIN') die('skip posix only test.');
-if (!function_exists('strftime')) die("skip, strftime not available");
```

Comment:
According to https://github.com/php/php-src/blob/master/ext/date/php_date.c#L415 this function is not always available.